resources/nuts-node/auth.js

```diff
@@ -1,18 +1,32 @@
 const config = require('../../util/config')
 var FormData = require('form-data')
 const apiHelper = require('./open-api-helper')
-const definitionLocation = 'https://raw.githubusercontent.com/nuts-foundation/nuts-auth/0cdd505d38ac3387062437ffe25863ca2cc4a11a/docs/_static/nuts-auth.yaml'
-const call = apiHelper.call({
+const specExperimental = 'https://raw.githubusercontent.com/nuts-foundation/nuts-node/master/docs/_static/auth/experimental.yaml'
+const specV0 = 'https://raw.githubusercontent.com/nuts-foundation/nuts-node/master/docs/_static/auth/v0.yaml'
+
+const callExperimental = apiHelper.call({
+  baseURL: `http://${config.nuts.node}`,
+  definition: specExperimental
+})
+const callV0 = apiHelper.call({
   baseURL: `http://${config.nuts.node}`,
-  definition: definitionLocation
+  definition: specV0
 })
 
 const auth = {
-  createLoginSession: async () => call('createSession', null, loginContract()),
-  createSession: async (contract) => call('createSession', null, contract),
-  sessionRequestStatus: async (id) => call('sessionRequestStatus', id),
-  validateContract: async (contract) => call('validateContract', null, contract),
-  createJwtBearerToken: async (context) => call('createJwtBearerToken', null, context),
+  drawUpContract: async (organisationDID) => callExperimental('drawUpContract', null, {
+    type: 'BehandelaarLogin',
+    language: 'NL',
+    version: 'v3',
+    legalEntity: organisationDID,
+  }),
+  createLoginSession: async (contract) => callExperimental('createSignSession', null, {means: 'irma', payload: contract.message, params: {}}),
+  createSession: async (contract) => callV0('createSession', null, contract),
+  sessionRequestStatus: async (id) => callExperimental('getSignSessionStatus', id),
```

Comment:
Why do some methods use the experimental version of the API while others don't? Are both APIs available on different endpoints on the same server operating on the same data? Will this not cause issues in the future?

Reply:
This will be merged into one v1 api soon

Reply:
👍🏻

Reply:
Fixed